templates/02-counter/src/main.js

```diff
@@ -19,9 +19,42 @@ function updateUI() {
     Array.from(document.querySelectorAll('.sign-in')).map(it => it.style = "display: block;");
   } else {
     Array.from(document.querySelectorAll('.after-sign-in')).map(it => it.style = "display: block;");
+    contract.getCounter().then(count => {
+      document.querySelector('#show').classList.remove('loader')
+      document.querySelector('#show').innerText = count == undefined ? "calculating..." : count
+      document.querySelector('#left').classList.toggle('eye')
+      Array.from(document.querySelectorAll('button')).map(it => it.disabled = false)
+    })
   }
 }
+// counter method
+var value = 1
```

Comment:
`var` is outdated syntax, ES6 code should use `let` (or `const` when applicable)